DependencyInjection/DoctrineExtension.php

```diff
@@ -429,6 +429,12 @@ protected function loadOrmEntityManager(array $entityManager, ContainerBuilder $
             ));
         }
 
+	    if (version_compare(Version::VERSION, "2.5.0-DEV") < 0) {
+		    $listenerId = sprintf('doctrine.orm.%s_listeners.attach_entity_listeners', $entityManager['name']);
+		    $listenerDef = $container->setDefinition($listenerId, new Definition('%doctrine.orm.listeners.attach_entity_listeners.class%'));
+            $listenerDef->addTag('doctrine.event_listener', array('event' => 'loadClassMetadata'));
+        }
```

Comment:
I don’t understand this part..using ORM 2.5.0 the listener is missing in the `attachToListener` method.